docs/source-fabric/fundamentals/precision.rst

```diff
@@ -29,25 +29,34 @@ This is how you select the precision in Fabric:
     # This is the default
     fabric = Fabric(precision="32-true")
 
-    # Also FP32
+    # Also FP32 (legacy)
```

Comment:
I'm wondering why we are calling this legacy now if it is just a shortcut/alias for true. I thought we decided to keep it. I would be fine also to reduce showcasing it in the docs as much since the options are now growing. It might look a bit overwhelming as there are many choices. 
In fact, this was my intention when I wrote the section, to show a few combinations but not literally every combination, since subsequent sections would mention the rest. So I suggest to keep this brief.